homeassistant/components/media_player/plex.py

```diff
@@ -217,7 +243,11 @@ def request_configuration(host, hass, config, add_devices_callback):
     def plex_configuration_callback(data):
         """Handle configuration changes."""
         setup_plexserver(
-            host, data.get('token'), hass, config, add_devices_callback)
+                host, data.get('token'),
+                True if data.get('has_ssl') else False,
```

Comment:
You'll need to validate the data. Just presence of it is not enough. Use `homeassistant.helpers.config_validation.boolean(value)` to convert the values.

Better would be if you implement checkbox support in the frontend https://github.com/home-assistant/home-assistant-polymer/blob/master/src/more-infos/more-info-configurator.html#L66-L73